### DISCUSSION TITLE
【臺灣正體注音符號方案】請問如何自定自己的發音？

### DISCUSSION THREAD

**Discussion by lunarythia** [rime/home#1626] [Q&A] [answered]
我發現雖然我用台灣的注音方案，但偶而還是找到只能用中國大陸的發音來輸入的字（大部分都OK，只有少數有問題）。

例如：
- 認識：只能打 ㄖㄣˋ ㄕ˙ (rèn **shi**)，不能用 ㄖㄣˋ ㄕˋ (rèn **shì**) 不然就會變「任事」
- 識：只能打 ㄕˊ (shí)，不能用 ㄕˋ (shì)
我之前也有找到其他有問題的字，但那時後沒想到來這裡問，所以我現在也找不到了。

請問要自定自己的發音該怎麼做？

（我是華僑所以我的中文可能會有點怪怪的，因為除了小時候的中文學校沒受過正式的中文教育）

**Comment by LEOYoon-Tsaw** [comment, marked as answer]
打開用戶檔案夾，在terra_pinyin.dict.yaml中修改即可。網上也可能有現成品，可以試著找找

**Reply by lunarythia** [reply]
如果 terra_pinyin 有新更新，我改的東西會不會就不見了？

**Reply by LEOYoon-Tsaw** [reply]
不要放入biuld就不會不見

**Reply by lunarythia** [reply]
了解了，謝謝！
